ExpensesSplitter.WebApi/Controllers/SettlementsController.cs

```diff
@@ -65,14 +65,39 @@ public ActionResult GetSettlementUsers(string id)
             List<SettlementUser> settlements = context.SettlementUsers.Where(x => x.SettlementId == id).ToList();
             var result = (from s in settlements
                          join u in context.Users on s.UserId equals u.Id
-                         select new { Id = u.Id, Name = u.Name, Email = u.Email, Login = u.Login, Group = s.GroupId, Role = s.RoleId}).ToList();
+                         join g in context.Groups on s.GroupId equals g.Id
+                         select new { Id = u.Id, Name = u.Name, Email = u.Email, Login = u.Login, Group = s.GroupId, GroupName = g.Name, Role = s.RoleId, SettlementUserId = s.Id}).ToList();
             if (result.Count != 0)
             {
                 return Ok(result);
             }
             return NotFound();
         }
         [HttpGet]
+        [Route("GetSettlementUsersWithoutAccount/")]
+        public ActionResult GetSettlementUsersWithoutAccount(string id)
```

Comment:
Chyba dokładnie to samo robi endpoint GET `/settlements/{settlementId}/users` (w `SettlementUsersController`), nie możesz go reużyć?